src/screens/Customers/AddCustomer.tsx

```diff
@@ -184,31 +205,32 @@ function AddCustomer(props: AddCustomerProps) {
           <div className={classes.textContainer}>
             <TextField
               label={'Meter Number'}
-              id={'meter-number'}
+              id={'meterNumber'}
               placeholder={'Input'}
               type="number"
-              value={meterNumber}
-              onChange={handleMeterNumberInput}
-              disabled={selectedMeterType === MeterType.INACTIVE || selectedMeterType === MeterType.NO_METER}
-              required={selectedMeterType !== MeterType.INACTIVE && selectedMeterType !== MeterType.NO_METER}
+              value={formik.values.meterNumber}
+              onChange={formik.handleChange}
+              error={formik.touched.meterNumber && Boolean(formik.errors.meterNumber)}
+              helperText={formik.touched.meterNumber && formik.errors.meterNumber}
+              disabled={formik.values.selectedMeterType === MeterType.NO_METER}
+              required={formik.values.selectedMeterType !== MeterType.NO_METER}
```

Comment:
I feel like it would make this cleaner from a UX perspective to just conditionally show/hide this field? I don't see why it needs to show in the disabled state.

Reply:
It might be marginally cleaner, but I'm going to keep it to match the Figma. If you feel strongly, we can address though.